src/components/Restoration/RestorationType.vue

```diff
@@ -1,56 +1,63 @@
 <template>
-  <v-card flat id="restoration-type" class="pa-6">
-    <v-row>
+  <div id="restoration-type">
+    <div class="section-container" :class="{ 'invalid-section': invalidSection }">
+      <v-row no-getters>
```

Comment:
"no-getters" gets rid of the weird negative margins, and means I don't need the "pa-6" you see in the old code on line 2.